requirements.txt

```diff
@@ -3,4 +3,5 @@ linux-metrics==0.1.4
 pint==0.7.2
 paho-mqtt==1.2
 aenum==1.4.5
-CoAPthon==4.0.2
\ No newline at end of file
+CoAPthon==4.0.2
+uptime==3.0.1
```

Comment:
I remembered when Christian collected this library requirement, Ravi's team member declared that in the future, we may need more libraries, and may ask again from legal team.
I'll send Christian an email to request that.